src/jit/morph.cpp

```diff
@@ -18116,19 +18116,19 @@ class LocalAddressVisitor final : public GenTreeVisitor<LocalAddressVisitor>
         if (m_compiler->verbose)
         {
             printf("LocalAddressVisitor visiting statement:\n");
-            m_compiler->gtDispTree(stmt);
+            m_compiler->gtDispTree(stmt->gtStmtExpr);
             m_stmtModified = false;
         }
 #endif // DEBUG
 
         WalkTree(&stmt->gtStmtExpr, nullptr);
 
-        // We could have somethinge like STMT(IND(ADDR(LCL_VAR))) so we need to escape
+        // We could have something like STMT(IND(ADDR(LCL_VAR))) so we need to escape
```

Comment:
Maybe change to "We could have a statement like IND(ADDR(LCL_VAR))" since STMT is no longer an oper.

Reply:
Fixed